Dev/config/strings.js

```diff
@@ -2,7 +2,7 @@
 export default {
 
     //---------------------  App info ---------------------------
-    AppTitle: 'Quran Connect',
+    AppTitle: 'QuranConnect',
```

Comment:
We should make a decision here

Reply:
Correct. I personally prefer the one word logo (like WhatsApp or LinkedIn, etc..) , but I'm open if you guys prefer to keep it as a 2 words logo.